README.md

```diff
@@ -51,13 +70,31 @@ We use semver on spaceduck so you can refer to a specific tag if you want instea
   <img  src="https://user-images.githubusercontent.com/32819563/111935037-3ab41d80-8a88-11eb-8797-2b6db14cbff8.png" alt="lualine replace mode screenshot">
 </center>
 
+## PACKER
+
 ```lua
 require("lualine").setup({
   options = {
     theme = "spaceduck",
   },
 })
 ```
+## LAZYVIM
+
+```lua
+return {
+    "nvim-lualine/lualine.nvim",
+    dependencies = { 'nvim-tree/nvim-web-devicons' },
+    lazy = false,
+    priority = 1000,
+    opts = {
+        options = {
+            globalstatus = true,
```

Comment:
I also use `globalstatus` as well but that's rather a personal preference. It's not mandatory for people to use Spaceduck so I'd rather remove it.